test/unit_tests/cryptography/CryptographyRepositorySpec.js

```diff
@@ -100,6 +100,56 @@ describe('CryptographyRepository', () => {
     });
   });
 
+  describe('getRemoteFingerprint', () => {
+    it('generates the remote fingerprint based on a prekey', async () => {
+      const database = TestFactory.storage_service.db;
+      await TestFactory.cryptography_repository.createCryptobox(database);
+      const userId = '6f656da7-0c52-44d1-959d-ddc9fbdca244';
+      const clientId = '689ce2df236eb2be';
+      const preKey = {
+        id: 3,
+        key:
+          'pQABAQMCoQBYIFycSfcOATSpOIkJz8ntEnFAZ+YWtzVaJ7RLeDAqGU+0A6EAoQBYIMEJnklbfFFvnFC41rmjDMqx6L0oVX5RMab3uGwBgbkaBPY=',
+      };
+      const fingerprint = await TestFactory.cryptography_repository.getRemoteFingerprint(userId, clientId, preKey);
+
+      expect(fingerprint).toEqual([
+        'c1',
+        '09',
+        '9e',
+        '49',
+        '5b',
+        '7c',
+        '51',
+        '6f',
+        '9c',
+        '50',
+        'b8',
+        'd6',
+        'b9',
+        'a3',
+        '0c',
+        'ca',
+        'b1',
+        'e8',
+        'bd',
+        '28',
+        '55',
+        '7e',
+        '51',
+        '31',
+        'a6',
+        'f7',
+        'b8',
+        '6c',
+        '01',
+        '81',
+        'b9',
+        '1a',
```

Comment:
Let's add an eslint exception here so that we can have the array in a single line